finetuner/classes/experiment.py

```diff
@@ -2,9 +2,9 @@
 
 from docarray import DocumentArray
 
-from finetuner.classes import Run
-from finetuner.client.client import Client
-from finetuner.constants import CONFIG, CREATED_AT, DESCRIPTION, NAME
+from ..client import FinetunerV1Client
+from ..constants import CONFIG, CREATED_AT, DESCRIPTION, NAME
+from .run import Run
```

Comment:
I dislike absolute imports in packages because the self-containment of sub-packages and the bottom-up approach is clearer with relative imports. For example in `finetuner.client.client` in order to subclass the base class I need to do `from finetuner.client.base import BaseClient` and then in `finetuner.client.__init__` in order to expose the main class of my subpackage I need to do `from finetuner.client.client import Client` when all these components are in the same sub-package. But its a personal preference. If you feel that absolute is clearer, its fine by me.

Reply:
I'd say let's stick to the absolute imports for now but be open to switching to relatives if it gets messy